### ISSUE TITLE
No way to obtain an AChecker API ID - is this project alive?

### ISSUE THREAD

**Issue by Nachasic** [cfpb/node-wcag#41]
CLI requires an AChecker API id:
```
$ wcag domainname.com
Please provide an AChecker API ID with `--id` or setting an `ACHECKER_ID` environment variable. Register at http://achecker.ca/register.php to get an ID.
```
`http://achecker.ca/register.php` returns 404, it looks like the original service is no longer there.

What is the status of this project? Any hope of getting it back to a useful state?

**Comment by anselmbradford** [comment]
Yeah, unfortunately achecker was shut down on April 30, 2021, and we removed our config related to this package from our site (https://github.com/cfpb/consumerfinance.gov/pull/6295, https://github.com/cfpb/consumerfinance.gov/pull/6670).

I'm not aware of anyone maintaining this repo or https://github.com/cfpb/node-wcag-cli 

It should probably be archived (cc @contolini)
